platform-sdk/swirlds-cli/src/main/java/com/swirlds/cli/logging/HtmlGenerator.java

```diff
@@ -0,0 +1,747 @@
+                .addClass(nodeStylingLabel)
+                .generateTag();
+        final String breakTag = new HtmlTagFactory("br", null, true).generateTag();
+
+        return inputTag + "\n" + labelTag + "\n" + breakTag + "\n";
+    }
+
+    /**
+     * Create a single checkbox filter
+     *
+     * @param elementName the name of the element to hide / show
+     * @return the checkbox
+     */
+    private static String createCheckboxFilter(@NonNull final String elementName) {
+        final String inputTag = new HtmlTagFactory("input", null, true)
+                .addClasses(List.of(FILTER_CHECKBOX_LABEL, elementName))
+                .addAttribute("type", "checkbox")
+                .addAttribute("checked", "checked")
+                .generateTag();
+
+        final String labelTag = new HtmlTagFactory("label", elementName, false).generateTag();
+        final String breakTag = new HtmlTagFactory("br", null, true).generateTag();
+
+        return inputTag + "\n" + labelTag + "\n" + breakTag + "\n";
+    }
+
+    /**
+     * Create a set of radio buttons that can hide elements with the given name
+     * <p>
+     * This method creates 3 radio buttons, whitelist, blacklist, and neutral
+     *
+     * @param elementName the name of the element to hide
+     * @return the checkbox
+     */
+    private static String createStandardRadioFilter(@NonNull final String elementName) {
+        final String commonRadioLabel = elementName + "-radio";
+
+        final String whitelistTag = new HtmlTagFactory("input", null, true)
+                .addClasses(List.of(FILTER_RADIO_LABEL, WHITELIST_LABEL, WHITELIST_RADIO_LABEL, elementName))
+                .addAttribute("type", "radio")
+                .addAttribute("name", commonRadioLabel)
+                .addAttribute("value", "1")
+                .generateTag();
+        final String neutralTag = new HtmlTagFactory("input", null, true)
+                .addClasses(List.of(FILTER_RADIO_LABEL, NEUTRALLIST_RADIO_LABEL, elementName))
+                .addAttribute("type", "radio")
+                .addAttribute("name", commonRadioLabel)
+                .addAttribute("checked", "checked")
+                .addAttribute("value", "2")
+                .generateTag();
+        final String blacklistTag = new HtmlTagFactory("input", null, true)
+                .addClasses(List.of(FILTER_RADIO_LABEL, BLACKLIST_LABEL, BLACKLIST_RADIO_LABEL, elementName))
+                .addAttribute("type", "radio")
+                .addAttribute("name", commonRadioLabel)
+                .addAttribute("value", "3")
+                .generateTag();
+
+        final String labelTag = new HtmlTagFactory("label", elementName, false).generateTag();
+        final String breakTag = new HtmlTagFactory("br", null, true).generateTag();
+
+        return whitelistTag + "\n" + neutralTag + "\n" + blacklistTag + "\n" + labelTag + "\n" + breakTag + "\n";
```

Comment:
Perhaps consider changing this utility to use a string builder instead of concatenation. Every time you concatenate two java strings, you essentially create an big array and copy the contents of two smaller arrays into it. Has the potential to get inefficient for large repeated append operations.